src/core/engines/Cesium/Feature/Raster/hooks.ts

```diff
@@ -150,6 +151,11 @@ export const useMVT = ({
         return currentTime === updatedAt.current;
       },
       style: (mvtFeature, tile) => {
+        const styleCacheKey = JSON.stringify(mvtFeature.properties);
```

Comment:
Properties in MVT feature is stringifyed json unless parsing properties, so I think it's almost no cost.